src/lib.rs

```diff
@@ -35,6 +35,8 @@
 //! };
 //! ```
 //!
+//! # Important Notes before compiling!
+//!
```

Comment:
I don't think this adds anything. To the contrary, it creates a section header with no content.

Reply:
```suggestion
```
Okay, better remove it.